src/test/java/com/wikia/webdriver/common/enums/Mercury.java

```diff
@@ -0,0 +1,27 @@
+package com.wikia.webdriver.common.enums;
+
+/**
+ * @ownership Content X-Wing Wikia
+ */
+public class Mercury {
```

Comment:
Class name called `Mercury` looks like bottomless pit.
Anyway if you apply my comment below, this class won't be needed anymore